### PR TITLE
Update spin.js w/ npm auto-update

### PR DESCRIPTION
**Pull request by klausenbusk**
The auto-update glob is out-of-date, so let's switch to NPM and fix the
glob.

Refs #302

---
This was done by-hand with help from `curl -s $(npm view X dist.tarball) | tar -tvzf -`

### CODE REVIEW

File: packages/s/spin.js.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/fgnass/spin.js.git",
+    "source": "npm",
+    "target": "spin.js",
     "fileMap": [
       {
         "basePath": "",
         "files": [
           "spin.js",
```
Comment (MattIPv4) [inline packages/s/spin.js.json:22]:
```suggestion
          "*.@(js|css|ts)",
```
There are also some typings included in the package, and as we support those, we might as well bring them in. [Glob](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2A.%40%28js%7Ccss%7Cts%29&matches=false&tests=%2F%2F%20Imported%20from%20spin.js%20NPM%20package&tests=spin.css&tests=spin.js&tests=package.json&tests=CHANGELOG.md&tests=LICENSE.md&tests=README.md&tests=spin.d.ts&tests=SpinnerOptions.d.ts)

File: packages/s/spin.js.json [hunk 2]
```diff
@@ -13,14 +13,14 @@
...
-          "spin.min.js"
+          "spin.css"
```
Comment (MattIPv4) [inline packages/s/spin.js.json:23]:
```suggestion
```
Will be made redundant by the above glob

File: packages/s/spin.js.json (hunk 1 above)
Comment (klausenbusk) [reply packages/s/spin.js.json:22]: Does people include `*.ts` directly on a webpage?

Comment (MattIPv4) [reply packages/s/spin.js.json:22]: Typings aren't used on web pages, but folks that are using Libman or Repl.it for example may want to be able to access typings. We support it, so we might as well include them if they're provided.

**Pull request merged**